### ISSUE TITLE
v1.0.17 - Only 1 Album, 1 Artist but over 5,000 Tracks

### ISSUE THREAD

**Issue by vk2him** [music-assistant/hass-music-assistant#142]
After updating to v1.0.17 it automatically started a scan - when it finished I had only 1 Album, 1 Artist but over 5,000 Tracks.

The only error in the logs were these normal ones:

```
Logger: music_assistant.mass.music.filesystem
Source: /usr/local/lib/python3.9/site-packages/music_assistant/controllers/music/providers/filesystem.py:376
First occurred: 11:11:31 AM (105 occurrences)
Last logged: 11:15:42 AM

nasfiles/Unknown Artist/Unknown Album/silence.mp3 is missing ID3 tags, use filename as fallback
nasfiles/Unknown Artist/Unknown Album/Guy Walks into a Bar.mp3 is missing ID3 tags, use filename as fallback
nasfiles/Unknown Artist/Unknown Album/ANW2050_03_Book-Of-Secrets.mp3 is missing ID3 tags, use filename as fallback
nasfiles/Unknown Artist/Unknown Album/Heart - Stairway to Heaven.m4a is missing ID3 tags, use filename as fallback
nasfiles/Unknown Artist/Unknown Album/Borat SMS.mp3 is missing ID3 tags, use filename as fallback

```

I then forced another scan during which it now found the "missing" Artists and Albums, however it created many extra log error entries. I'll log a new Issue re that as it may or may not be related. This issue is specifically that the initial scan after upgrade lost the previous Albums and Artists - could it be the race issue again?

**Comment by marcelveldt** [comment]
This is definitely the race issue (again/still) because the missing artists and albums come back f you trigger the rescan (reload integration) if HA is already running. It is the side effect I was talking about that a shell script runs to mount your network drive.

**Comment by marcelveldt** [comment]
@vk2him can you confirm this is resolved in 1.0.18 ?

**Comment by vk2him** [comment]
Fixed in new version - thanks
